BambeeHR/pollen
# addIcon/PRIVA-1290 - Add New Request Icon for Navigation

## Issue(s)
Adding Request Icon for App navigation.

## Description
This Icon is to be used in APP for Requests.
![Screenshot_20220616_040707](https://user-images.githubusercontent.com/10406785/174057259-7ec4f98c-2d5d-4787-a558-f8f3e32028a3.png)

## Screenshot(s)
In Storybook (BaseIcon):
![Screenshot_20220616_040628](https://user-images.githubusercontent.com/10406785/174057171-3a7a920f-9ad8-4427-8bd2-90126ac288a1.png)

In Storybook (HighlightedIcon):
![Screenshot_20220616_040751](https://user-images.githubusercontent.com/10406785/174057410-ca2cd74c-8b27-4c04-b7e1-88c571adb0a2.png)

## Testing
1. npm run storybook
2. Navigate to BaseIcon and see request Icon.
3. Navigate to HighlightedIcon and see request Icon.

## Risk Assessment
LOW
